src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/H2Translator.java

```diff
@@ -21,12 +21,14 @@
 import com.feedzai.commons.sql.abstraction.engine.DatabaseEngineRuntimeException;
 import com.feedzai.commons.sql.abstraction.engine.OperationNotSupportedRuntimeException;
 import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Lists;
 import org.apache.commons.lang3.StringUtils;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import static com.feedzai.commons.sql.abstraction.dml.dialect.SqlBuilder.select;
```

Comment:
I'm missing the purposes of this imports

Reply:
I forgot to remove unneeded imports